### ISSUE TITLE
[version=2.7]Class 'PhpAmqpLib\Exception\AMQPConnectionClosedException' not found

### ISSUE THREAD

**Issue by Guojunzhou-git** [php-amqplib/php-amqplib#763]
I use version2.7, but receive an "Class Not Found Error for AMQPConnectionClosedException"
```
PHP Fatal error:  Uncaught Error: Class 'PhpAmqpLib\Exception\AMQPConnectionClosedException' not found in /var/www/nb70plm8791289asja/api.caidouenglish.com/vendor/php-amqplib/php-amqplib/PhpAmqpLib/Wire/IO/StreamIO.php:299
Stack trace:
#0 /var/www/nb70plm8791289asja/api.caidouenglish.com/vendor/php-amqplib/php-amqplib/PhpAmqpLib/Connection/AbstractConnection.php(349): PhpAmqpLib\Wire\IO\StreamIO->write('\x01\x00\x00\x00\x00\x00\v\x00\n\x002\x00\x00\x00\x00...')
#1 /var/www/nb70plm8791289asja/api.caidouenglish.com/vendor/php-amqplib/php-amqplib/PhpAmqpLib/Connection/AbstractConnection.php(470): PhpAmqpLib\Connection\AbstractConnection->write('\x01\x00\x00\x00\x00\x00\v\x00\n\x002\x00\x00\x00\x00...')
#2 /var/www/nb70plm8791289asja/api.caidouenglish.com/vendor/php-amqplib/php-amqplib/PhpAmqpLib/Channel/AbstractChannel.php(240): PhpAmqpLib\Connection\AbstractConnection->send_channel_method_frame(0, Array, Object(PhpAmqpLib\Wire\AMQPWriter))
#3 /var/www/nb70plm8791289asja/api.caidouenglish.com/vendor/php-amqplib/php-amqplib/PhpAmq in /var/www/nb70plm8791289asja/api.caidouenglish.com/vendor/php-amqplib/php-amqplib/PhpAmqpLib/Wire/IO/StreamIO.php on line 299
```

**Comment by Guojunzhou-git** [comment]
In version 2.7.3, actually there is no AMQPConnectionClosedException file Exactly...
![image](https://user-images.githubusercontent.com/10252607/70782267-8d048a00-1dc1-11ea-9421-f7255f1b3b85.png)

**Comment by ramunasd** [comment]
AMQPConnectionClosedException is available since version 2.8.0, previous versions do not throw it. Looks like You somehow mixed files from different versions.
